### DISCUSSION TITLE
Chain analysis option for coins in escrow

### DISCUSSION THREAD

**Discussion by coupes-privilegie0a** [bisq-network/bisq#6139] [Ideas]
First I want to stress that I have no idea of the feasibility of this idea.

Buyer could have the option to ask coins in escrow to be submitted to chain analysis, potentially accepting to pay for it if this required a paid API.

This could allow seller to advertise that their coins are clean or non-KYC, thus increasing transparency and allowing the Bisq community to push back against money laundering in a decentralized, anonymous and virtuous way. I suppose sellers ready to submit their coins to chain analysis could reasonably expect a premium for their good faith.

Technically, this could either use a built-in tool that would need to be developed for Bisq and paid for by DAO, or an API for some third party chain analysis service, potentially requiring payment.

**Comment by w0000000t** [comment]
This can make sense, and at a same time be a sensible topic, that might collide with Bisq's own policy for which kyc is the illicit activity, and chanalysing coins could be seen as sort of prying into users' private information?
Other than this, since market needs to be free, it can also be argued that users should be free to decide whether they expect a certain "kind" of coins, and they are available to pay more for it

**Reply by coupes-privilegie0a** [reply]
I can definitely see how this could be seen as a sensitive topic, especially if chain analysis becomes a de facto standard that forces unwilling sellers to lower their asking price. In a way, it *is* a push against fungibility, I’m just not sure this could have ever been avoided due to the public nature of the ledger.

Regarding the privacy implications, nothing is currently preventing a buyer to subject the coins he just bought to chain analysis, so I don’t see how the proposal worsens this situation. Whatever is disclosed would have been know eventually, the only difference is that there could either be some sort of penalty if what is disclosed is not conform to what had been agreed upon, or a way to cancel the trade with minimal fee.

But maybe I’m failing to account for some scenarios!

**Comment by MwithM** [comment]
Bisq, together with mixers, is an essential tool for Bitcoin's fungibility.
Inviting users to use a third party tool to judge if a coin is good enough to be traded through arbitrary rules would be a fatal step. Inviting users to comply with other rules than the ones enforce by Bitcoin protocol, like AOPP proposers and wasabi did is completely against the principles of Bitcoin and Bisq.
Basically, at Bisq, like at the Bitcoin network, all bitcoins are equal.

Market is free, I think there are enough KYC alternatives out there for those who believe that panopticon is good for society.

**Reply by coupes-privilegie0a** [reply]
I totally understand this view. I wish such a tool would not be necessary, as I really don’t want a panopticon either. I actually hope it will never be necessary. Alas attempts to establish a database of private wallets might make it necessary.

The KYC alternatives are just that: KYC. This system can also be seen as a pushback against KYC and centralized exchanges.
